Source/Bridge.swift

```diff
@@ -74,28 +74,26 @@ public final class Bridge: Bridgable {
 //        let replyMessage = message.replacing(data: data)
 //        callBridgeFunction("send", arguments: [replyMessage.toJSON()])
 //    }
-
-    /// Evaluates javaScript string directly as passed in sending through the web view
-    func evaluate(javaScript: String, completion: CompletionHandler? = nil) {
+    @MainActor
+    @discardableResult
+    func evaluate(javaScript: String) async throws -> Any? {
```

Comment:
We can return a non-optional `Any` from this method. I've addressed this in https://github.com/hotwired/strada-ios/pull/30.